etc/profile-m-z/mutt.profile

```diff
@@ -34,14 +36,77 @@ noblacklist ${HOME}/sent
 blacklist /tmp/.X11-unix
 blacklist ${RUNUSER}/wayland-*
 
+include allow-perl.inc
+include allow-python.inc
+
 include disable-common.inc
 include disable-devel.inc
+include disable-exec.inc
 include disable-interpreters.inc
 include disable-passwdmgr.inc
 include disable-programs.inc
+include disable-xdg.inc
 
+mkfile ${HOME}/.elinks
+mkfile ${HOME}/.emacs
+mkfile ${HOME}/.mailcap
+mkfile ${HOME}/.msmtprc
+mkfile ${HOME}/.muttrc
+mkfile ${HOME}/.nanorc
+mkfile ${HOME}/.signature
+mkfile ${HOME}/.vimrc
+mkfile ${HOME}/.viminfo
+mkfile ${HOME}/.vimrc
+mkfile ${HOME}/.w3m
+mkdir ${HOME}/.Mail
```

Comment:
Considering `etc/templates/profile.template`, shouldn't `mkdir` come
before `mkfile`?

Reply:
profile.template has it alphabetical (with two lines there're only two options) which is not bad for small list. With longer lists like here it's fine in my opinion to have `noblacklist`, `mk{dir,file}` and `whitelist` listed the files/dirs in the same order on all three.

Reply:
> profile.template has it alphabetical (with two lines there're only two
> options) which is not bad for small list. With longer lists like here it's
> fine in my opinion to have `noblacklist`, `mk{dir,file}` and `whitelist`
> listed the files/dirs in the same order on all three.

I mean, the order of files/dirs would still be the same in both lists, the only
difference when sorting from the first `mkfile` to the last `mkdir` is that the
`mkdir` entries would come before the `mkfile` ones.  Having it all sorted
means being able to add/edit anything anywhere in the range of [`mkdir`,
`whitelist`] and then just run `:sort u` on it to make it all sorted, without
having to visually scan things (also, creating dirs before files seems a bit
more natural to me).  Anyways, these were sorted by @bbhtt, but there's a few
others; I'll add a few code suggestions to demonstrate; feel free to
accept/reject.